src/main/tut/1.9-function-types.md

```diff
@@ -0,0 +1,142 @@
+```Haskell
+eval :: ((a⇒b) × a) -> b
+```
+```scala
+def eval[A, B]: (`A⇒B` × A) => B
```

Comment:
it should be:
```scala
(A `⇒` B)
```
not: 
```scala
`A⇒B`
```
The latter case means a type of kind `*` whose name is `A⇒B`, the former a type of kind `* -> * ->` named `⇒` an parameterised by `A` and `B`. You also need to add the parentheses I _think_

Reply:
Yeah, that is what I had before, but in the book the object name is "a⇒b", and it didn't fell right to have  (A `A⇒B` B) or change the name. Anyways, it is much easier to read, so I will revert it back.